model/feature-flag/events.yaml

```diff
@@ -1,14 +0,0 @@
-groups:
-  - id: feature_flag
```

Comment:
one of identical feature_flag events has to go so that we can have one group id that follows the pattern.

Removing it also partially unblocks  https://github.com/open-telemetry/weaver/issues/306